java/robox-slicer-extension/robox-slicer-flow/src/main/java/com/roboxing/slicerextension/flow/Slicer.java

```diff
@@ -122,13 +124,13 @@ public void postProcess(File inputGCode, File resultGCode) throws IOException{
                         // String newLine = strLine.replace(m.group(0), String.format(Locale.ROOT, "E%.5g%n", newExtrusion));
                         String newLine = strLine.replace(m.group(0), "E"+(double)Math.round(newExtrusion * 100000d) / 100000d);
                         // System.out.println("new Line :" + newLine);
-                        outputRelative.write(newLine+"\n");
+                        outputRelative.write(newLine + newLine);
```

Comment:
Gosh, I missed that one!  Good catch.  Feel free to update to a constant (static final String)